src/scss/initial/_colors.scss

```diff
@@ -19,13 +19,9 @@
 ///   Accoutrement Variables
 /// @link https://www.oddbird.net/cascading-colors/
 ///   Cascading Color Systems
-/// @group init-colors
-
```

Comment:
Removing this switches the order of the heading and the map. Should we add this back in?

![Screen Shot 2022-08-09 at 3 30 10 PM](https://user-images.githubusercontent.com/552316/183745257-922af98f-8ed1-4274-b2e6-73f3093d6a27.jpg)